crates/storage-query-datafusion/src/invocation_state/schema.rs

```diff
@@ -14,6 +14,8 @@ use crate::table_macro::*;
 
 use datafusion::arrow::datatypes::DataType;
 
+define_sort_order!(sys_invocation_state(partition_key, id));
```

Comment:
Does this statement say that the invocation status table is sorted by partition key and then id? If yes, how are we ensuring the id sort order? It looks as if we are only sorting wrt to the partition key.